script.js

```diff
@@ -13,20 +13,20 @@ function getComputerChoice() {
 // console.log(getComputerChoice());
 
 // Write logic to get human choice
-// Create function getHumanChoice
+// Create function gethumanSelection
 // Function should return valid choice based on user input
 // Test that the function returns what's expected
 
 // Refactor to remove else if statements
-function getHumanChoice() {
-  let choice = prompt("Enter either rock, paper or scissor!");
-  choice = choice.toLowerCase();
-  if (choice == "rock" || choice == "paper" || choice == "scissor") {
-    return choice;
-  } else return "incorrect input";
-}
+// function gethumanSelection() {
+  // let choice = prompt("Enter either rock, paper or scissor!");
+  // choice = choice.toLowerCase();
+  // if (choice == "rock" || choice == "paper" || choice == "scissor") {
+  //   return choice;
+  // } else return "incorrect input";
+// }
 // debug log
-// console.log(getHumanChoice());
+// console.log(gethumanSelection());
```

Comment:
Eventually you should remove this code from here. It holds little value being commented in this file.

If you want to keep it around for learning purpose, consider creating another JS file for this, like `learning-dump.js`, or adding sections in the `README.md` that explains specific points.

Git keeps track of the code at all commits, so if you need to go back in time to look at old code, you should use Git (and not comments).